tools/rna_tools/rnaz/.shed.yml

```diff
@@ -1,5 +1,5 @@
 name: rnaz
-owner: bgruening
+owner: rnateam
```

Comment:
to keep the tool-linage we should keep the old name

Reply:
can you please also adopt this file to look similar to this: https://github.com/galaxyproject/tools-iuc/blob/57493fdc88e6128bed127993464f2a6480e2774d/tool_collections/bamtools/bamtools_split/.shed.yml

This will create a bunch of single-tool-repos in the toolshed.